schema_generator/__init__.py

```diff
@@ -155,6 +155,35 @@ def checkbox_property(question):
     }}
 
 
+def checkbox_tree_property(question):
+    """
+    Convert a checkbox tree question into JSON Schema by flattening the tree structure
+    """
+    def flatten(options):
+        for option in options:
+            yield option
+            for option in flatten(option.get('options', [])):
+                yield option
+
+    # items may not be unique, so using a set not a list
```

Comment:
Yes it will, see https://github.com/alphagov/digitalmarketplace-content-loader/blob/checkbox-tree/dmcontent/questions.py#L500